pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -190,6 +190,8 @@ type AmazonVPCNetworkingSpec struct {
 	// The container image name to use, which by default is:
 	// 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.5
 	ImageName string `json:"imageName,omitempty"`
+	// Env is a list of environment variables to set in the container.
+	Env []EnvVar `json:"env,omitempty"`
```

Comment:
I think that `Environment` might be preferable to `EnvVar`, thoughts

Reply:
When we discussed this during office hours we decided to follow upstream regarding environment variable terminology:

https://github.com/kubernetes/api/blob/d5ab0ddd8b7334701d758c13524fec4830b4c923/core/v1/types.go#L1826-L1827